stubs/Core/Utility/ExtensionManagementUtility.php

```diff
@@ -47,4 +47,19 @@ public static function loadExtLocalconf(bool $false): void
     public static function extRelPath(string $key): void
     {
     }
+
```

Comment:
I would remove the docblocks here.

Reply:
I have slimmed them down to only the type-annotations, as not all parameters have native typehints yet